posts/20230223/index.qmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "Reviewing a case study about real estate market prediction"
+author: "Athula Pudhiyidath"
+date: "2023-02-23"
+image: "This image will show on the main page (this is optional).jpg"
+description: A discussion about better practices for binary classification.
+categories: [paper]
+---
+
+:::{.callout-note}
+- Paper: [*"Machine Learning Approaches to Real Estate Market Prediction Problem: A Case Study*](https://arxiv.org/pdf/2008.09922.pdf)
+- Presenter: [Athula](https://www.linkedin.com/in/athula-pudhiyidath/)
+- Attendees: [Hannah](https://www.linkedin.com/in/hannah-peeler-a4751bb2/), [Hongsup](https://www.linkedin.com/in/hongsupshin/),[Kate](https://www.linkedin.com/in/kate-behrman/)
+:::
+
+## Why this paper
+In my work, I deal with real estate data. The data consists of various attributes of real estate properites, some of which which follow the ebbs and flows of the mutable economy. I came across this paper while perusing arxiv, and I was compelled by its abstract: using real estate data that spanned 10 years from a single county in Florida, the research compares four ML models that make predictions about whether or not a property’s appraised price matches its eventually listed price. Since the data spans such a long time, the researchers accounted for fluctuations in socio-economic factors over time in the model; this facet was of particular interest to me to conceptualize my own work and I was interested to learn about the authors incorporated these factors in machine learning algorithms to make predictions about the real estate market.
+
+While this abstract seemed reasonably promising to understand the application of ML in real estate evaluation, when I read the paper more closely, I found myself questioning many of the approaches the authors took to answer their central question. Therefore, this blog is structured such that it summarizes the authors’ work as outlined in this paper, and then also offers alternative considerations from the perspective of a reviewer.
+
+## The data  
+The authors wanted to build and compare models for predicting home prices. They posit that previous literature addressed home pricing with hedonic regression models, i.e. linear regression models with specific focus on property outcomes like its individual environmental characteristics, but not necessarily accounting for broader socio-economic factors. With the modeling conducted in this paper, the authors attempted to model home pricing accounting for such factors. However, instead of predicting home pricing as a continous variable as with regression, the authors chose to make home prediction a binary classification problem. The data used in this research consist of ~94,000 rows of publicly available real estate sale data from a single county in Florida, Volusia County.
```

Comment:
I think some people might not be familiar with "hedonic regression" (a person like me!). Is this just a vanilla linear regression but people are calling it differently in real estate domain, or is it something else?

Reply:
I've added some clarification about this